From d4d71ae2d604f215a687d931aa76ff7923ffa60f Mon Sep 17 00:00:00 2001
From: Jon TURNEY <jon.turney@dronecode.org.uk>
Date: Wed, 19 Nov 2014 11:32:43 +0000
Subject: [PATCH 24/24] Disable timeout waiting for crash reporting server

15s is not enough if we are going to ask user if they want make a dump etc.

Perhaps I need to change my server implementation to do the dump, then ask
if should be uploaded etc. ?

Signed-off-by: Jon TURNEY <jon.turney@dronecode.org.uk>
---
 src/client/windows/crash_generation/crash_generation_client.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/client/windows/crash_generation/crash_generation_client.cc b/src/client/windows/crash_generation/crash_generation_client.cc
index 1e86dce..9204feb 100644
--- a/src/client/windows/crash_generation/crash_generation_client.cc
+++ b/src/client/windows/crash_generation/crash_generation_client.cc
@@ -36,7 +36,7 @@ namespace google_breakpad {
 
 const int kPipeBusyWaitTimeoutMs = 2000;
 
-#ifdef _DEBUG
+#if 1
 const DWORD kWaitForServerTimeoutMs = INFINITE;
 #else
 const DWORD kWaitForServerTimeoutMs = 15000;
-- 
2.1.1

